federator/src/lib/FederatorNFT.ts

```diff
@@ -212,7 +212,17 @@ export class FederatorNFT {
 
         const { blockHash, transactionHash, logIndex, blockNumber } = log;
 
-        const { _to: receiver, _from: sender, _tokenId: tokenId } = log.returnValues;
+        const {
+          _to: receiver,
+          _from: sender,
+          _tokenId: tokenIdStr,
+          _originChainId: originChainIdStr,
+          _destinationChainId: destinationChainIdStr,
+        } = log.returnValues;
+
+        const originChainId = Number(originChainIdStr);
+        const destinationChainId = Number(destinationChainIdStr);
+        const tokenId = Number(tokenIdStr);
```

Comment:
Token Id can be bigger than number, use bignumber or treat it as a string

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/286/commits/e1eaee1f5e99eafc950a5a90b9c2d3dfd3d9ac9f